app/models/event.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+
+class Event < ApplicationRecord
+  validates :name, presence: true
+  validates :location, presence: true
+  validates :day, presence: true
+  validates :description, presence: true
+
+  belongs_to :creator, class_name: 'User'
```

Comment:
In comparison to the code above, we specified class name here because the association name (creator) is different from the class name (user).